docs/basics.txt

```diff
@@ -550,9 +550,9 @@ IMPORTANT: Do not use bin/janusgraph.sh.  That starts the default configuration,
 
 
 
-=== JanusGraph Server as a REST-style Endpoint
+=== JanusGraph Server as a HTTP-style Endpoint
```

Comment:
HTTP is an endpoint. Do we have to say HTTP style or can we just say "HTTP Endpoint"